server/game/gameSystems/DelayedEffectSystem.ts

```diff
@@ -43,18 +43,11 @@ export class DelayedEffectSystem<TContext extends AbilityContext = AbilityContex
         //     properties.ability = event.context.ability;
         // }
 
-        const { title, when, duration, limit, immediateEffect, ...otherProperties } = properties;
-
-        const renamedProperties = { ...otherProperties, ongoingEffect:
-            OngoingEffectLibrary.delayedEffect({
-                title,
-                when,
-                immediateEffect,
-                limit
-            }) };
-
         const delayedEffectSource = event.sourceCard;
 
+        const renamedProperties = event.renamedProperties;
+        const duration = properties.duration;
```

Comment:
Looks like we're still using `properties` here - the goal is to remove the `generatePropertiesFromContext` call completely and put everything relevant on the event itself